CMakeLists.txt

```diff
@@ -9,17 +8,15 @@ set(CMAKE_MODULE_PATH
 # include commands for coloring prints
 include(colors)
 
+# include
+include(version_parse)
+
```

Comment:
please move this include under the # versioning comment below

Reply:
Done